spec/models/lettings_log_spec.rb

```diff
@@ -1454,6 +1454,15 @@ def previous_postcode_fields
         expect(record_from_db["vacdays"]).to eq(0)
         expect(lettings_log["vacdays"]).to eq(0)
       end
+
+      it "correctly derives and saves first_time_property_let_as_social_housing" do
+        record_from_db = ActiveRecord::Base.connection.execute(
```

Comment:
why do we hand craft SQL queries when we have active record?

```suggestion
record_from_db = LettingsLog.find(id: lettings_log.id)
```